### PR TITLE
Glass Classification

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Glass Classification

- The aim of this project is to classify the glass type using percentage of minerals present in each class of glass

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [ ] Contributor

Closes: #469 

### Describe the add-ons or changes you've made

In this project we have performed a analysis and visualization of the training dataset with different Exploratory Data Analysis techniques. Then a comparative analysis of different ML Classifiers have been done to predict type of glass. After performing the comparative analysis of the classifiers, we can see that the Random Forest Classifier gives the best results with a score of 83.72%. However to get better accuracy we build a Keras Sequential model which gives a validation accuracy of 95.24% and Testing accuracy of 90.91%.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Excellent choice of project. Only change it needs is in .iynb files, the file names should be as below. File names should be as "EDA_and_ML_classifier_model.ipynb" and "Keras_model.ipynb". Kindly do these changes. @Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
I have done the required changes. Please check @prathimacode-hub .

**Review by prathimacode-hub — APPROVED** [review]
As usual nothing to say, your work is exquisite and outstanding. Wonderful choice of the project.  🤩🤩✔@Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
> As usual nothing to say, your work is exquisite and outstanding. Wonderful choice of the project. 🤩🤩✔@Soumayan-pal01

Thanks @prathimacode-hub
